src/Network/Wai/SAML2/Request.hs

```diff
@@ -113,8 +114,7 @@ renderXML AuthnRequest{..} =
     ,   documentEpilogue = []
     }
     where
-        timestamp = T.pack $
-            formatTime defaultTimeLocale timeFormat authnRequestTimestamp
+        timestamp = T.pack $ iso8601Show authnRequestTimestamp
```

Comment:
I am wondering if it would make sense to change the definition of `timeFormat` in https://github.com/mbg/wai-saml2/blob/3c933d783097772fc8f428ea05a62792ad9b8bec/src/Network/Wai/SAML2/XML.hs#L75 to better suit this (either by changing the `String` value there or by making it an alias for something that's exported from `Data.Time.Format.ISO8601`)? 

It would be good to avoid having different parts of the library use different definitions of time formats. Arguably, the existing definition of `timeFormat` is probably wrong and I should've just used definitions from `Data.Time.Format.ISO8601` throughout to begin with?

Reply:
Since this is not used anywhere other than `parseUTCTime`, I suppose we could just use `iso8601ParseM` (done).

Reply:
Awesome, thank you!